pcsx2/gui/MainFrame.cpp

```diff
@@ -802,18 +802,18 @@ void MainEmuFrame::ApplyCDVDStatus()
 
 	switch (Source)
 	{
-		case CDVD_SourceType::Iso:
-			label = _("Boot ISO");
-			break;
-		case CDVD_SourceType::Disc:
-			label = _("Boot CDVD");
-			break;
-		case CDVD_SourceType::NoDisc:
-			label = _("Boot BIOS");
-			break;
-		default:
-			label = _("Boot BIOS");
-			break;
+	case CDVD_SourceType::Iso:
```

Comment:
Are you sure the formatting on the switch case is correct?

Reply:
Fixed it